.github/workflows/test.yml

```diff
@@ -53,36 +53,61 @@ jobs:
           macos-13,
           windows-latest,
         ]
+        wasmer-feature: [
+          wasmer_sys,
+          wasmer_wamr
+        ]
+        exclude:
+          # Our intended target for wasmer_wamr feature is iOS and Android.
+          # For now, wasmer_wamr tests are only run on macos-latest.
+          # Additionally, building with wasmer_wamr feature flag on windows is not currently working.
+          # See https://github.com/holochain/holochain-wasmer/issues/117
+          - os: windows-latest
+            wasmer-feature: wasmer_wamr
+          - os: macos-13
+            wasmer-feature: wasmer_wamr
+          - os: ubuntu-latest
+            wasmer-feature: wasmer_wamr
     steps:
       - name: checkout
         uses: actions/checkout@v4
 
       - name: 'Common Test Setup Actions'
         uses: ./.github/actions/common-pre
 
-      - name: build windows
+      - name: install build dependencies (macos-latest | macos-13, wasmer_wamr)
+        if: (matrix.os == 'macos-latest' || matrix.os == 'macos-13') && matrix.wasmer-feature == 'wasmer_wamr'
+        shell: bash
+        run: brew install cmake
+    
+      - name: install build dependencies (ubuntu-latest, wasmer_wamr)
+        if: matrix.os == 'ubuntu-latest' && matrix.wasmer-feature == 'wasmer_wamr'
```

Comment:
Is this to support running wamr tests on Ubuntu later? I'd be open to doing that now if it works but it would be good to leave a comment here with the intention while this is excluded in the matrix

Reply:
Yes tests on ubuntu work -- it was @neonphog's suggestion to only run them on macos-latest to limit the ci workload where ios/android builds are the prioritized target. I'd lean towards running on all platforms as well but appreciate that slow ci can be a burden. Happy to do whatever you both decide!

Reply:
Yeah that is fair, I agree that testing on one platform is sensible then

Reply:
I should also formulate a question or a request =) 
` if: matrix.os == 'ubuntu-latest' && matrix.wasmer-feature == 'wasmer_wamr'`
This will currently never be entered, because according to the matrix `wasmer_wamr` is only used for `macos-latest`. It seems we should therefore change this to say
` if: matrix.os == 'macos-latest' && matrix.wasmer-feature == 'wasmer_wamr'`

Reply:
Right - I removed the workflow steps that will never run.